### ISSUE TITLE
Exercise about heading tags <h1> ... <h6>, accidentally refers to header tags <header>

### ISSUE THREAD

**Issue by michaelsndr** [freeCodeCamp/freeCodeCamp#43427]
## Affected page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-size-of-a-header-versus-a-paragraph-tag.md

https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/adjust-the-size-of-a-header-versus-a-paragraph-tag

## Expected behavior

In the title and the first paragraph, header tags `<header>` are mentioned, but the exercise is about heading tags `<h1> ... <h6>`.

**Comment by Anurag5086** [comment]
I'm working on fixing this!

**Comment by ieahleen** [comment]
@Anurag5086 thank you for your entusiasm, this issue has not been triaged yet. Look at the issues with label "help wanted" to know what needs help.

**Comment by Anurag5086** [comment]
> @Anurag5086 thank you for your entusiasm, this issue has not been triaged yet. Look at the issues with label "help wanted" to know what needs help.

Oh Okay! Thank you!

**Comment by Sembauke** [comment]
Thank you for bringing this up @michaelsndr,

I do agree that this is not correct as the header tag refers to some other HTML element.

In the challenge linked, we can change the header tag to heading (or plural) headings.
We should also look if there are other challenges that have the same.

@Anurag26  thanks for the quick response, but I will still like to hear the opinions of others.

**Comment by ieahleen** [comment]
checked the curriculum with `grep` on `header` string and careful scrolling, and found these files

curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-size-of-a-header-versus-a-paragraph-tag.md
curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/link-to-internal-sections-of-a-page-with-anchor-elements.md **(the example calls an `h1` element a `header`)**
/workspace/freeCodeCamp/curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-markdown-previewer.md **calls h1 to h6 headers**
curriculum/challenges/english/03-front-end-development-libraries/react/compose-react-components.md
curriculum/challenges/english/03-front-end-development-libraries/react/create-a-controlled-form.md
curriculum/challenges/english/03-front-end-development-libraries/react/create-a-react-component.md
curriculum/challenges/english/03-front-end-development-libraries/react/render-state-in-the-user-interface-another-way.md
curriculum/challenges/english/03-front-end-development-libraries/react/render-state-in-the-user-interface.md
curriculum/challenges/english/03-front-end-development-libraries/react/set-state-with-this.setstate.md
curriculum/challenges/english/03-front-end-development-libraries/sass/store-data-with-sass-variables.md

Do we want to change header->heading in all these cases when talking about `h1` to `h6` elements?

**Comment by Sembauke** [comment]
Let's see what others think of it

**Comment by ShaunSHamilton** [comment]
Both the HTML spec, and Mozilla refer to them as _"heading elements"_. So, I suggest we stick with that.

We should also take care with mentioning `tags` versus `elements`.
- This is a tag: `<h1>`
- This is an element: `h1`

**Comment by Sembauke** [comment]
@Anurag5086, are you up for the task?

**Comment by Anurag5086** [comment]
@Sembauke Yeah sure. If you don't mind could you just brief me once what have we finally decided on? Would be really helpful! Thanks!

**Comment by ieahleen** [comment]
@Anurag5086 the changes mentioned here, making sure that the `h1` to `h6` elements are called headings, not headers, and check that `h1` is correctly called element, and `<h1>` is correctly called tag and if necessary change that

**Comment by Anurag5086** [comment]
@ieahleen @Sembauke Yeah sure, sounds good! Working on it.

**Comment by Nzamuyejoram** [comment]
@ieahleen  Yeah cool. i really like that spirit ! Working on it

**Comment by ieahleen** [comment]
@Nzamuyejoram this issue has already a linked PR which is almost done, I wouldn’t advise you to work on it

**Comment by Nzamuyejoram** [comment]
@ieahleen . yeah cool. noted
